spec.html

```diff
@@ -24617,7 +24619,11 @@ <h1>PerformEval ( _x_, _callerRealm_, _strictCaller_, _direct_ )</h1>
       <emu-clause id="sec-hostensurecancompilestrings" aoid="HostEnsureCanCompileStrings">
         <h1>HostEnsureCanCompileStrings ( _callerRealm_, _calleeRealm_ )</h1>
         <p>The host-defined abstract operation HostEnsureCanCompileStrings takes arguments _callerRealm_ (a Realm Record) and _calleeRealm_ (a Realm Record). It allows host environments to block certain ECMAScript functions which allow developers to compile strings into ECMAScript code.</p>
-        <p>An implementation of HostEnsureCanCompileStrings may complete normally or abruptly. Any abrupt completions will be propagated to its callers. The default implementation of HostEnsureCanCompileStrings is to unconditionally return an empty normal completion.</p>
+        <p>An implementation of HostEnsureCanCompileStrings must conform to the following requirements:</p>
+        <ul>
+          <li>It must propagate throw completions to its callers.</li>
+        </ul>
+        <p>The default implementation of HostEnsureCanCompileStrings is to unconditionally return NormalCompletion(~empty~).</p>
```

Comment:
The word "unconditionally" isn't used consistently for the return values modified in this patch. I believe it's included merely for emphasis, but I wonder if consistency would be valuable.

Reply:
Good idea, consistency would be valuable. I don't think we need to include "unconditionally". Any preference here?

Reply:
Omitting it is my preference, too